### ISSUE TITLE
Accessibility test top 10 page views

### ISSUE THREAD

**Issue by eelzi-ONRR** [DOI-ONRR/onrr.gov-site#2734]
Trusted Tester and lighthouse

**Comment by eelzi-ONRR** [comment]
See #2666 for mroe details

**Comment by eelzi-ONRR** [comment]
See #2666 for mroe details

**Comment by lpgoldstein** [comment]
![image.png](https://images.zenhubusercontent.com/5c59d3cd7592186396262002/64aca068-b58c-44d0-8c27-f984c4f03809)

**Comment by lpgoldstein** [comment]
dentify your agency's top ten viewed publicly available internet pages within the reporting period. Test each page for full conformance to Section 508 (this will require comprehensive manual testing in addition to automated or a hybrid test process if utilized in your agency). For each page listed, report the number of failures by Section 508 standard/WCAG Success Criterion. Report the number of times each SC failed by page. Provide page name, page description and URL. (*Required)
Identify your agency's top ten viewed electronic documents posted on your public website within the reporting period. Test each document for full conformance to Section 508 (this will require comprehensive manual testing in addition to automated or a hybrid test process if utilized in your agency). For each document, report the number of failures by Section 508 standard/WCAG Success Criterion. Please also provide the document name, document type, and document description.
